syntax = "proto3";
package user;

service UserService {
  rpc GetUser (UserRequest) returns (UserReply);
}

message UserRequest {
  int32 user_id = 1;
}

message UserReply {
  string user_name = 1;
  string email = 2;
  bool is_active = 3;
}